packages/app/src/components/Me/ProfileImageSettings.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
```

Comment:
There are 2 files with same name and different extention exist. (.jsx and .tsx)
Please move the code in this file to .tsx file but you do not need to typescriptize in this PR

Please create a new task and typescriptize the moving code.

Reply:
I Just removed this file ProfileImageSettings.jsx